client/src/components/feedback.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { Container } from 'react-bootstrap'
+import FeedbackSend from './feedback-send';
+import FeedbackComment from './feddback-comment'
+import axios from 'axios'
+
+class Feedback extends Component{
+ 
+
+    state = {
+        allFedback: [],
+        pieceFedback: [],
+        maxCommentShow:1,
+        start:0,
+        last:0
+
+    }
+
+    onClickShowMoreComment(){
+
+        this.setState((prevState)=>{
+            return {
+                pieceFedback: prevState.allFedback.slice(prevState.start, prevState.last+prevState.maxCommentShow),
+                last: prevState.last+prevState.maxCommentShow
+            }
+        });
+    }
+
+
+    componentDidMount() {
+
+        axios.get('http://localhost:8080/api/feedback/2')
+            .then((response) => {
+                this.setState({
+                    allFedback: response.data,
+                    pieceFedback: response.data.slice(this.state.last, this.state.maxCommentShow),
+                    last: this.state.maxCommentShow
+                })
+            });
+    }
+
+
+    render(){
+
+        return (
+
+            <Container className="containerFeedback">
+                <FeedbackSend />
+                {
+
+                    this.state.pieceFedback.map((item, index) => {
+                        return (
+                      
+                                <FeedbackComment item={item} key={index} />
```

Comment:
Using index as a key is not good practice. Check this [link](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318), please.